2. Web API now calls Microsoft Graph/TodoListService/Controllers/TodoListController.cs

```diff
@@ -97,9 +97,9 @@ public async Task<string> CallGraphApiOnBehalfOfUser()
                 dynamic me = await CallGraphApiOnBehalfOfUser(accessToken);
                 return me.UserPrincipalName;
             }
-            catch (MsalUiRequiredException ex)
+            catch (MicrosoftIdentityWebChallengeUserException ex)
```

Comment:
maybe we should catch both for now.
cc: @jmprieur

Reply:
we could raise an issue to fix this, but i think we leave it for now. it's already been included since 2.2.1-preview, and requires some thought + test coverage.